lib/vmdb/util.rb

```diff
@@ -1,7 +1,10 @@
 module VMDB
   module Util
-    def self.http_proxy_uri
-      proxy = VMDB::Config.new("vmdb").config[:http_proxy] || {}
+    def self.http_proxy_uri(proxy_config = :default)
+      # TODO: (julian) This looks really messy. Need to make it look nicer.
+      vmdb_proxy = VMDB::Config.new("vmdb").config[:http_proxy]
```

Comment:
@juliancheal can you update this one?  `VMDB::Config.new("vmdb").config` ~= `Settings`.